10up/ads-txt
# Add PHP version check for plugin.

### Description of the Change


Add minimum version check for the plugin before loading it. This will ensure plugin update doesn't break the site that don't match our minimum PHP version.

<img width="1239" alt="image" src="https://github.com/10up/ads-txt/assets/10358350/14d18658-732d-4817-ada4-7791d9c4feba">


Closes #148

### How to test the Change


1. Setup a WordPress environment running PHP 7.3
2. Install ads.txt plugin by downloading it with this [feature](https://github.com/10up/ads-txt/archive/refs/heads/fix/148-php-ver-check.zip) branch.
3. Verify that plugin functionality doesn't load and you see an admin error message for php version. ( refer above screenshot in Description )

### Changelog Entry


> Fixed - Better error handling for environments that don't match our minimum PHP version

### Credits


Props @dkotter, @rahulsprajapati 

### Checklist:
